routes/api.php

```diff
@@ -21,11 +21,12 @@
     return $request->user();
 });
 
-Route::post('login', [AuthController::class, 'login']);
-Route::post('register', [AuthController::class, 'register']);
+Route::post('/login', [AuthController::class, 'login']);
+Route::post('/register', [AuthController::class, 'register']);
 
 Route::middleware('auth:sanctum')->group(function () {
     Route::apiResource('categories', ProjectCategoryController::class);
+    Route::post('/users/update/{user}', [AuthController::class,'update']);
```

Comment:
why post instead of put/patch for the update of a user: 

is this not enough: Route::put("/users", [AuthController::class, "update"]); Since you're using route model binding i don't think {user} is necessary